psydac/feec/multipatch/examples_nc/h1_source_pbms_nc.py

```diff
@@ -0,0 +1,322 @@
+"""
+    solver for the problem: find u in H^1, such that
+
+      A u = f             on \\Omega
+        u = u_bc          on \\partial \\Omega
+
+    where the operator
+
+      A u := eta * u  -  mu * div grad u
+
+    is discretized as  Ah: V0h -> V0h  in a broken-FEEC approach involving a discrete sequence on a 2D multipatch domain \\Omega,
+
+      V0h  --grad->  V1h  -—curl-> V2h
+"""
```

Comment:
Is this example tested in the CI?

Reply:
See `psydac/feec/multipatch/tests/test_feec_poisson_multipatch_2d.py` .